addons/l10n_ec/models/account_move.py

```diff
@@ -48,7 +48,7 @@
     "04": [
         'ec_dt_04',
         'ec_dt_05',
-        'ec_dt_18',
+        'ec_dt_18',  # TODO 18
```

Comment:
@pepetreshere should we add `ec_dt_01` in these lists, remove `ec_dt_18`, both ?
:confused:

Reply:
Hello @cormaza may I ask you for your help here?
We really don't know why we added so many DOC Types and what was the purpose behind them back in the time. In this case, we are struggling to understand what was/is the purpose of the DOC Type (18) as it seems not necessary and we can just use (01) instead both for vendor bills and customer invoices.

Also, I have found [this legal reference](https://www.sri.gob.ec/DocumentosAlfrescoPortlet/descargar/fb95cafc-a8ca-4a4c-afb6-12c4153165f0/FICHA%20TECNICA%20COMPROBANTES%20ELECTR%C3%93NICOS%20ESQUEMA%20OFFLINE.pdf) only for DOC Types, but I couldn't find the list of the other ones or their legal/commercial purpose.

![image](https://user-images.githubusercontent.com/45669815/193331688-3517952c-c747-4dc9-9754-3a2b26f51c65.png)


Any insights on this will help a lot!
Thanks!

cc' @pepetreshere @jco-odoo @odoostan

Reply:
You can refer to this documentation, it's related with ATS

https://www.sri.gob.ec/o/sri-portlet-biblioteca-alfresco-internet/descargar/72d717c2-88ed-47b7-baba-50b87b7198b7/Ficha%20Tecnica%20Transaccional%20Simplificado%20ATS.pdf

You can refer to table 4, this is like voucher type, It is a bit confusing because you use a code for sales and another for purchases, but they refer to the invoice, in this case 18 is more for invoices, 01 is for bills

![image](https://user-images.githubusercontent.com/1674751/193335151-9a5d23f5-7bfa-44e8-a89a-bf7d06fd5df9.png)

![image](https://user-images.githubusercontent.com/1674751/193335207-6ba01abf-185f-4944-b9f7-00ee1d09e42b.png)

![image](https://user-images.githubusercontent.com/1674751/193335258-db40a6b1-a17a-458d-90b9-06acc8cdcd28.png)

Reply:
@cormaza thank you very much for the info!
So, using Doc Type 01 or 18 will be important for the ATS report mainly, as for the usage, if you use 01 for both sides (Customer Invoices & Bills) it will be possible, then probably another logic can be implemented to add it in the ATS or not, as if the Purchase WTH is created then you don't need it in the ATS report, as far as I remember (talking about purchase withholding ATS 2.0).

What we are trying to see, is if we can avoid having the Doc Type 18 and use the 01 intead.

cc' @pepetreshere @jco-odoo @odoostan

Reply:
I'm not sure what you want to use that code for, I suppose it has to be for the issuance of electronic documents, in that case you could hardcode in some way if you have limited the type of documents you are going to issue, that is, only invoices, credit notes, debit notes, liquidation of purchase, the other codes are more used for purchases but I suppose that it can affect the new retention that has more integration with the ATS, my advice would be to hardcode the necessary data for electronic emission where necessary, as a map between ATS codes(this is the data you have in l10n_ec) and codes used in electronic emission

Reply:
@vbe-odoo I think we should just use code 01 either for the sale invoice or purchase invoice. The MVP focus is electronic invoicing, ATS will not be covered in this release. And even for the ATS, it will be easier to do the mapping just for the ATS report than doing it everywhere else (the same is true for reimbursements). I think this is the same suggestion from @cormaza .

Reply:
Hi, @pepetreshere @cormaza thanks both for your insights on this point.

So, @odoostan , as a conclusion - in order to resolve this comment if you agree - the idea here will be to replace `ec_dt_18` with `ec_dt_01` and add this last one (`ec_dt_01`) where it is missing in other parts.

And then we do the mapping when working on the ATS report (after the MVP release).

Thanks!
cc' @fvz-odoo @ren-odoo @jco-odoo